src/api/spec/models/token/workflow_spec.rb

```diff
@@ -135,15 +135,18 @@
       let(:payload) { github_payload_closed }
       let(:workflows_yml_file) { File.expand_path(Rails.root.join('spec/support/files/workflows.yml')) }
       let(:downloader) { instance_double(Workflows::YAMLDownloader) }
+      let(:target_project) { create(:project, name: 'home:Iggy:test-project:PR-4') }
 
       before do
+        target_project
```

Comment:
```suggestion
      let!(:target_project) { create(:project, name: 'home:Iggy:test-project:PR-4') }

      before do
```
Using `let!` prevents from calling `target_project` in the `before` block.

Reply:
true